tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/timeline/timeline_module.ts

```diff
@@ -13,16 +13,43 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 
+import {
+  BrowserModule,
+  HAMMER_GESTURE_CONFIG,
+  HAMMER_LOADER,
+} from '@angular/platform-browser';
+import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
 import {CommonModule} from '@angular/common';
 import {NgModule} from '@angular/core';
+import {GestureConfig} from '@angular/material';
+import {MatButtonModule} from '@angular/material/button';
+import {MatSliderModule} from '@angular/material/slider';
 
 import {ExecutionDataModule} from '../execution_data/execution_data_module';
 import {TimelineComponent} from './timeline_component';
 import {TimelineContainer} from './timeline_container';
 
 @NgModule({
   declarations: [TimelineComponent, TimelineContainer],
-  imports: [CommonModule, ExecutionDataModule],
+  imports: [
+    BrowserAnimationsModule,
+    BrowserModule,
+    CommonModule,
+    ExecutionDataModule,
+    MatButtonModule,
+    MatSliderModule,
+  ],
+  // TODO(cais): The following two providers are meant to make the mat-slider
+  // dragging to work. But the dragging is still not working, needs more
+  // investigation. May be related to:
+  // https://github.com/angular/components/issues/7905
+  providers: [
+    {
+      provide: HAMMER_LOADER,
+      useValue: () => new Promise(() => {}),
+    },
+    {provide: HAMMER_GESTURE_CONFIG, useClass: GestureConfig},
+  ],
```

Comment:
If HAMMER_LOADER does not help, would it make sense to just omit it for now and wait til Angular 9? 

I strongly recommend just using http://go/mdn/HTML/Element/input/range until we are ready.

Reply:
Done.